### DISCUSSION TITLE
Why did you replace alpine with buster-slim?

### DISCUSSION THREAD

**Discussion by salos1982** [uyamazak/hc-pdf-server#288] [Q&A] [answered]
Hello? I do not understand why did you replace apline with buster-slim as base docker image? Are there any issues with alpine and puppeteer?

**Comment by uyamazak** [comment, marked as answer]
Thank you for comment! please see below PR.

https://github.com/uyamazak/hc-pdf-server/pull/156

The main reason is the final image size.

I just tried to build both with the latest main branch on my PC (macOS Big Sur), and the results were about the same.
```
$ docker build -t hc-pdf-server:debian .

$ docker build -t hc-pdf-server:alpine -f Dockerfile.alpine .

$ docker images                         
REPOSITORY                 TAG          IMAGE ID       CREATED          SIZE
hc-pdf-server              alpine       d926a43ed121   30 seconds ago   868MB
hc-pdf-server              debian       3f56d45ea460   5 minutes ago    808MB
```

Another reason is that I have the impression that Debian is less problematic for applications with many dependencies and complexities such as Chrome, which is required by Puppeteer.

https://superuser.com/questions/1219609/why-is-the-alpine-docker-image-over-50-slower-than-the-ubuntu-image

**Comment by uyamazak** [comment]
The following example from the official documentation also shows the node-slim image first. 

https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker

There's also a minor difference: Debian is `google-chrome-stable` and Alpine is `chromium-browser`.

I haven't had any particular problems with alpine, but such reliability was one of the deciding factors for me.
